### DISCUSSION TITLE
Achievements Flow Not Working

### DISCUSSION THREAD

**Discussion by Ankitz007** [lowlighter/metrics#1494] [General]
Hi. Please replace the following in your action workflow file for achievements workflow -
` - uses: lowlighter/metrics@latest`

to this - 
`- uses: lowlighter/metrics@96dde98a034ab75fec7884f9161bc12759275d36`

FYI, it's the hash for the commit after which @lowlighter merged the workaround after recent changes in github search. (#1487)
Whenever the new release is published, maybe you can revert it back.

Thanks and cheers.

**Comment by Vault108** [comment]
Switching branches to master will also fix it
[#1479](https://github.com/lowlighter/metrics/issues/1479)

**Comment by lowlighter** [comment]
It should now work again on `@latest` again now 👍

**Reply by Ankitz007** [reply]
Thank you Simon!
